spec/classes/pulpcore_spec.rb

```diff
@@ -51,6 +51,29 @@
           is_expected.to contain_exec('pulpcore-manager reset-admin-password --random')
         end
 
+        it 'does not install Redis' do
+          is_expected.not_to contain_class('redis')
+        end
+
+        it 'does not configure Pulpcore connection to Redis' do
+          is_expected.to contain_concat__fragment('base')
+            .without_content(/REDIS_HOST/)
+            .without_content(/REDIS_POST/)
+            .without_content(/REDIS_DB/)
+        end
+
+        it 'does not configure content caching' do
+          is_expected.to contain_concat__fragment('base')
+            .without_content(/CACHE_ENABLED = True/)
+            .without_content(%r{CACHE_SETTINGS = \{\n    'EXPIRES_TTL': 60,\n\}})
+        end
```

Comment:
This is already covered by https://github.com/theforeman/puppet-pulpcore/pull/207/files#diff-257fe3136ecfdbfcb123b6445ff2de6c21d1fe08b0d6af54c08dde40637d2594R26

Reply:
Nice catch, thank you.